protocol/v2/ssv/runner/aggregator.go

```diff
@@ -529,3 +547,51 @@ func constructVersionedSignedAggregateAndProof(aggregateAndProof spec.VersionedA
 
 	return ret, nil
 }
+
+// isAggregator returns true if the signature is from the input validator. The committee
+// count is provided as an argument rather than imported implementation from spec. Having
+// committee count as an argument allows cheaper computation at run time.
+//
+// Spec pseudocode definition:
+//
+//	def is_aggregator(state: BeaconState, slot: Slot, index: CommitteeIndex, slot_signature: BLSSignature) -> bool:
+//	 committee = get_beacon_committee(state, slot, index)
+//	 modulo = max(1, len(committee) // TARGET_AGGREGATORS_PER_COMMITTEE)
+//	 return bytes_to_uint64(hash(slot_signature)[0:8]) % modulo == 0
+func isAggregator(targetAggregatorsPerCommittee uint64, committeeCount uint64, slotSig []byte) bool {
+	modulo := committeeCount / targetAggregatorsPerCommittee
+	if modulo == 0 {
+		// Modulo must be at least 1.
+		modulo = 1
+	}
+
+	b := hashSha256(slotSig)
+	return binary.LittleEndian.Uint64(b[:8])%modulo == 0
+}
+
+var sha256Pool = sync.Pool{New: func() interface{} {
+	return sha256.New()
+}}
```

Comment:
I think it's probably better if this wasn't "static" so we would create it inside the aggregator runner since it's using it. but I'm not stubborn on this change since this was like this just in another package before.